packages/widgets/src/blocks/legacy-widget/editor.scss

```diff
@@ -4,7 +4,7 @@
 	}
 	background: $white;
 	border-radius: $radius-block-ui;
-	border: 1px solid $gray-900;
+	border: $border-width solid $gray-900;
 	padding: $grid-unit-15 - 1px; // Subtract the border width.
```

Comment:
This is a place where the variable can actually be helpful in the math. You can even remove the CSS comment if you do this:

`padding: $grid-unit-15 - $border-width;`